### PR TITLE
Add intro to Procedure in the Hive connector

### PR DESCRIPTION
**Pull request by rosewms**
## Description

Add an introduction to the Procedure section in the Hive connector. Include information about call a procedure and an example.

> Is this change a fix, improvement, new feature, refactoring, or other? Improvement

> Is this a change to the core query engine, a connector, client library, or the SPI interfaces? Hive connector

> How would you describe this change to a non-technical end user or system administrator?

## Documentation

( ) No documentation is needed.
(x) Sufficient documentation is included in this PR.
( ) Documentation PR is available with #prnumber.
( ) Documentation issue #issuenumber is filed, and can be handled later.

## Release notes

(x) No release notes entries required.
( ) Release notes entries required with the following suggested text:

### CODE REVIEW

File: docs/src/main/sphinx/connector/hive.rst [hunk 1]
```diff
@@ -956,6 +956,13 @@ The following operations are not supported when ``avro_schema_url`` is set:
 Procedures
 ----------
 
+Use the :doc:`/sql/call` procedure to perform data manipulation or
```
Comment (mosabua) [inline docs/src/main/sphinx/connector/hive.rst:959]: This should say

Use the CALL statement to perform 


or

Call a procedure to

**Review by Ordinant — APPROVED** [review]
I had a suggested edit, but version two here is much better than mine!

File: docs/src/main/sphinx/connector/hive.rst (hunk 1 above)
Comment (mosabua) [reply docs/src/main/sphinx/connector/hive.rst:959]: or even 

CALL a procedure to

Comment (rosewms) [reply docs/src/main/sphinx/connector/hive.rst]: I like the first one, PR is updated to reflect the edit.

**Review by mosabua — APPROVED** [review]

**Review by hashhar — COMMENTED** [review]
In my opinion it'd be better to add explicit examples for each of the procedures instead of this summary.

File: docs/src/main/sphinx/connector/hive.rst [hunk 2]
```diff
@@ -956,6 +956,13 @@ The following operations are not supported when ``avro_schema_url`` is set:
 Procedures
 ----------
 
+Use the :doc:`/sql/call` statement to perform data manipulation or
+administrative tasks, such as flushing the metadata caches::
+
+    CALL system.flush_metadata_cache()
```
Comment (findepi) [inline docs/src/main/sphinx/connector/hive.rst:962]: I don't think this procedure should be spotlighted as the first example.
I'd rather have users not need to use it.

Comment (electrum) [reply docs/src/main/sphinx/connector/hive.rst:962]: You could use a fake name here. `system.example_procedure()`

Comment (rosewms) [reply docs/src/main/sphinx/connector/hive.rst]: Updated to a more generic example.

File: docs/src/main/sphinx/connector/hive.rst [hunk 3]
```diff
@@ -956,6 +956,13 @@ The following operations are not supported when ``avro_schema_url`` is set:
 Procedures
 ----------
 
+Use the :doc:`/sql/call` statement to perform data manipulation or
+administrative tasks, such as flushing the metadata caches::
```
Comment (mosabua) [inline docs/src/main/sphinx/connector/hive.rst:960]: this sentence is now wrong..

**Review by mosabua — CHANGES_REQUESTED** [review]
Please update..

**Review by mosabua — APPROVED** [review]

**Comment by electrum** [comment]
Nit: don't end commit title in period: https://cbea.ms/git-commit/#seven-rules

**Comment by rosewms** [comment]
> Nit: don't end commit title in period: https://cbea.ms/git-commit/#seven-rules

@electrum fixed

File: docs/src/main/sphinx/connector/hive.rst [hunk 4]
```diff
@@ -956,6 +956,13 @@ The following operations are not supported when ``avro_schema_url`` is set:
 Procedures
 ----------
 
+Use the :doc:`/sql/call` statement to perform data manipulation or
```
Comment (electrum) [inline docs/src/main/sphinx/connector/hive.rst:959]: We should mention that the procedure needs to be qualified with the catalog name. For example, if the catalog is named `hive`, the call should be
```sql
CALL hive.system.example_procedure()

File: docs/src/main/sphinx/connector/hive.rst [hunk 5]
```diff
@@ -956,6 +956,13 @@ The following operations are not supported when ``avro_schema_url`` is set:
 Procedures
 ----------
 
+Use the :doc:`/sql/call` statement to perform data manipulation or 
+administrative tasks. Procedures must include a qualified connector name::
+
+    CALL hive.system.example_procedure()
```
Comment (hashhar) [inline docs/src/main/sphinx/connector/hive.rst:962]:
```suggestion
administrative tasks. Procedures must include a qualified catalog name. For example if your Hive catalog is called `web`::

    CALL web.system.example_procedure()
```

**Pull request merged**